### ISSUE TITLE
location of scripts

### ISSUE THREAD

**Issue by hdiamondlowe** [zkbt/mosasaurus#23]
Would it make more sense to move the scripts folder out of the mosasaurus folder and into the same place where all the working directories are?

**Comment by zkbt** [comment]
Yes -- absolutely! I think we should have one worked example somewhere that's easy to copy from, but we should totally get rid of the messy (and now kind of inconsistent) scripts in the directory.

**Comment by hdiamondlowe** [comment]
So, apparently this just works if you move the `scripts` folder into the data directory, which in retrospect makes sense. For my LDSS3C data of LHS3844b, this looks like:

```
LHS3844 --> LDSS3C --> data --> ut20190619_20 --> *.fits
                            --> scripts --> lhs3488_base.py
                                        --> lhs3844_ut190619.py
```

I went ahead and moved my LHS3844 scripts from the mosasaurus scripts directory into a local one where my data is. I did the same for my LHS1140, GJ1132, and K2-25 spcripts, but I'll leave the rest there for @zkbt to decide what to do with. I agree that one should be left in there as an example.
